src/taipy/rest/api/schemas/datanode.py

```diff
@@ -64,6 +64,13 @@ class SQLDataNodeConfigSchema(DataNodeConfigSchema):
     write_table = fields.String()
 
 
+class MongoCollectionDataNodeConfigSchema(DataNodeConfigSchema):
+    db_username = fields.String()
```

Comment:
do we not expose `port`, `host`, `db_name` or `collection_name`, or we can access them in the properties? If we can access them from the `properties`, then do we need `db_username`, `db_password`, etc.?